### PR TITLE
Adding responsiveness to the buttons in homepage. 

### PR DESCRIPTION
**Pull request by eddierayo-156**
## Description

Please read the [Contribution Guidelines](../CONTRIBUTING.md) before opening a pull request.

### Summary

The Devfolio Apply and Discord buttons have been modified to have both images and the text inside the button. Used media queries to manipulate the format and position of the buttons in accordance with the screen size of an iPhone X show in the posted screenshot. Had a bit of trouble dealing with the apply button as it had both an image and text to combine. 

### Browser checklist

This PR has been tested in the following browsers:

- [x] Chrome
- [ ] Safari
- [ ] Firefox

### Closes issue #32 

iphone X mobile

### CODE REVIEW

**Comment by Aniket762** [comment]
hey @eddierayo-156 can you please resolve the conflicts and send the PR again?

**Comment by eddierayo-156** [comment]
I believe I added the updated file correctly. Please let me know if it did do so. Also, I forgot to take into account for landscape mode so I added that functionality. 

iphoneX portrait mobile

Before Landscape:
mobile problem
After:
iphoneX landscape-1
iphoneX landscape-2

File: src/pages/home.jsx [hunk 1]
```diff
@@ -351,8 +351,8 @@ function home2() {
         </Content>
         <Date src={require("../assets/images/date.png").default} />
          <Buttons>
-          <Button1 class = "Apply">
-           <Apply src={require("../assets/images/devfolio.png").default} />
+          <Button1>
+           <Register src={require("../assets/images/devfolio.png").default} />
```
Comment (Bartek532) [inline src/pages/home.jsx:355]: but where is `alt` attribute? :P

File: src/pages/home.jsx [hunk 2]
```diff
@@ -196,33 +201,58 @@ export const Date = styled.img`
...
    margin-left: 20px;
    background-color: #0360DE;
    margin-top:-5%;
+   cursor: pointer;
+
+
+   @media only screen and (width: 360px){
+    height: 20px;
+    width: 120px;
+    padding: 15px;
```
Comment (Bartek532) [inline src/pages/home.jsx]: I think paddings in `rem` or `em` are better option

File: src/pages/home.jsx [hunk 3]
```diff
@@ -196,33 +201,58 @@ export const Date = styled.img`
...
+    margin-right: -10%;
+    padding-bottom:2-8px;
```
Comment (Bartek532) [inline src/pages/home.jsx]: wait, what?

File: src/pages/home.jsx [hunk 4]
```diff
@@ -254,7 +303,10 @@ function home2() {
         </Content>
         <Date src={require("../assets/images/date.png").default} />
          <Buttons>
-          <Button1><Register src={require("../assets/images/devfolio.png").default} />Apply</Button1>
+          <Button1 class = "Apply">
+           <Apply src={require("../assets/images/devfolio.png").default} />
```
Comment (Bartek532) [inline src/pages/home.jsx]: `alt`

File: src/pages/home.jsx [hunk 5]
```diff
@@ -243,25 +267,41 @@ export const Date = styled.img`
...
+   @media only screen and (min-width: 769px) and (max-width: 936px) {
+     display: auto;
+     margin-right: -120%;
+   }
+   @media only screen and (min-width: 937px) and (max-width: 1024px) {
+     display: auto;
+   }
+   @media only screen and (min-width: 1025px) and (max-width: 1200px) {
+     display: auto;
+   }
```
Comment (Bartek532) [inline src/pages/home.jsx:304]: there are really so many media queries needed?

**Comment by sumana2001** [comment]
@eddierayo-156 Please look into the reviews given. It'll be great if you could send the pr again after resolving the conflicts.

**Review by sumana2001 — COMMENTED** [review]
@eddierayo-156 Please look into the reviews given. It'll be great if you could send the pr again after resolving the conflicts.

**Pull request closed without merging**